guidelines/outline.md

```diff
@@ -0,0 +1,123 @@
+# Style guide Language (ab-CD)
+
+<!-- toc -->
+
+	Insert your locale name into the line below and leave it in your style guide unless it does not apply to your locale. You may translate it if your style guide is in your language
+
+This style guide provides instruction for translating to the [Your locale name] locale. This style guide is to be used in coordination with the [General Mozilla L10n Style Guide](mozilla_general/README.md).
+
+	Include other relevant introductory information here, such as a community webpage or contact information.
+
+## Formality and Tone
+
+	Who is the target user for this project and what is their background?
+
+	How would a target user for this project expect to interact with this project?
+
+	Is formal language appropriate for all of your language's Mozilla l10n projects, or only some of them? Which ones?
+
+	Is informal language appropriate for all of your language's Mozilla l10n projects, or only some of them? Which ones?
+
+## Natural expression
+
+	What kind of expression sounds natural in your locale?
+
+	Does your locale allow for many borrowed words and phrases or few?
+
+## Handling cultural references, idioms, and slang
+
+	How are cultural references, idioms, and slang handled in your locale?
+
+	Are there resources for cultural references, idioms, and slang in your language?
+
+## Units and Unit Conversion
+
+	Does your locale use the metric system or the imperial system?
+
+### Date Format
+
+	How are the date formats for weeks and months expressed? Erase the comment markers around the example template or show the format with your own example.
+
+
+    | Name | Format | Example |
+    | ---- | ------ | ------- |
+    | Short | mm/dd/yy | 12/31/99 |
+    | Abbreviated | mmm dd | Dec 31 |
```

Comment:
Do we have needs with single letter Month representation?
| Abbreviated | m dd | D 31 |

Reply:
I'm not sure how common that is?